src/gui/plotWidget.h

```diff
@@ -134,30 +142,50 @@ class Plot : public QWidget {
      * @brief Disable the auto updating timer.
      */
     void disableUpdating();
+
     /**
      * @brief Delete all selected graphs.
      */
     void deleteSelectedGraphs();
 
    private:
     /**
-     * @brief Clear the current selection insde the plot.
+     * @brief Clear the current selection inside the plot.
      */
     void clearSelection();
 
+    /**
+     * @brief Update the plot after a unit change was detected
+     *
+     * Calculation is done in two steps. First from the current unit to kN
+     * and afterwards from kN to the next unit. This reduces the amount of
+     * different factors.
```

Comment:
That's not really true, the calculation is done in one step. This explanation makes it seem that this operation takes a long time. Also, I think it's better to let this be an implementation detail.
```suggestion
     * To go from the current unit to the `next`, all values in the plot are multiplied by a conversion factor.
```